offchain-modules/src/packages/db/btc.ts

```diff
@@ -59,4 +69,18 @@ export class BtcDb {
       take,
     });
   }
+
+  async getLockRecordsByUser(userAddr: string): Promise<LockRecord[]> {
+    return await this.connection.manager.query(
+      `select btc.sender as sender, ckb.recipient_lockscript as recipient , btc.amount as lock_amount,ckb.amount as mint_amount,btc.txid as lock_hash FROM btc_lock btc join ckb_mint ckb on btc.txid = ckb.id where btc.sender = ?`,
```

Comment:
why use raw sql instead of TypeORM way?

Reply:
not sure about the compatibility between different database with raw SQL

Reply:
okay, got it. I will update the code